R/get_bw_matrix.R

```diff
@@ -40,7 +43,8 @@ get_bw_matrix.BigWigFileList <- function(bw, regions,
 #' @importFrom GenomeInfoDb seqlevels seqlevels<-
 #' @importFrom rtracklayer summary
 get_bw_matrix.BigWigFile <- function(bw, regions,
-                          type = c("mean", "min", "max", "coverage", "sd")){
+                          type = c("mean", "min", "max", "coverage", "sd"),
+                          by = NULL){
   
   type <- match.arg(type, choices = c("mean", "min", "max", "coverage", "sd"))
```

Comment:
oh just btw, I was being a goofus when I wrote this line, you don't have to pass `choices` to `match.arg` if the variable is prespecified in the function definition. This can just be `type <- match.arg(type)` and it will check against the vector in the function definition for valid matches. Thought I'd point that out so you don't learn this anti-pattern from my silly mistake.